package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-super-confluence",
   "description": "A hubot script for searching and browsing Confluence.",
-  "version": "0.2.1",
+  "version": "0.2.3",
```

Comment:
Ok done. I also changed it so that errors from wiki search aren't shown when auto-searching based on questions overheard on chat.